### PR TITLE
Increase shell inertia coverage 

### PR DESCRIPTION
**Pull request by dschwen**
Refs #15067

### CODE REVIEW

File: modules/tensor_mechanics/test/tests/shell/dynamics/shell_dynamics_bending_moment.i [hunk 1]
```diff
@@ -43,262 +43,232 @@
...
-    family = LAGRANGE
-  [../]
-  [./rot_y]
-    order = FIRST
-    family = LAGRANGE
-  [../]
+  [disp_x]
+  []
+  [disp_y]
+  []
```
Comment (recuero) [inline modules/tensor_mechanics/test/tests/shell/dynamics/shell_dynamics_bending_moment.i:49]: Cryptic use of defaults.

**Review by recuero — DISMISSED** [review]

File: modules/tensor_mechanics/test/tests/shell/dynamics/shell_dynamics_bending_moment.i (hunk 1 above)
Comment (dschwen) [reply modules/tensor_mechanics/test/tests/shell/dynamics/shell_dynamics_bending_moment.i:49]: I removed those as they are IMO the most basic thing and clutter up the input considerably. But I can undo this change. I have to fix the SQA error anyways.

Comment (recuero) [reply modules/tensor_mechanics/test/tests/shell/dynamics/shell_dynamics_bending_moment.i:49]: I am fine with it.

Comment (dschwen) [reply modules/tensor_mechanics/test/tests/shell/dynamics/shell_dynamics_bending_moment.i]: Oh. well, i rolled it back already. Maybe best to stick to a small change set anyways. I now see that you actually approved the PR with the comment above. This is not visible in the github webinterface at all (only saw that in my email). The comment sounded more like you want this changed.

Comment (recuero) [reply modules/tensor_mechanics/test/tests/shell/dynamics/shell_dynamics_bending_moment.i]: Among the many lines of input file needed to set up that problem, one main piece of information should be the element type. Not sure if I agree that the default option here is the most basic thing. It's one common finite element option that happens to be default. 

Somehow the approval got stale. Not sure how.

File: modules/tensor_mechanics/test/tests/shell/dynamics/tests [hunk 2]
```diff
@@ -1,32 +1,43 @@
 [Tests]
-  design = "C0TimoshenkoBeam.md"
-  issues = "#15067"
   [verification_tests]
-    requirement = "The mechanics system shall accurately predict the"
-                  " dynamic behavior of a shell element with clamped-"
-                  "free-free-free conditions when "
-    [./shell_dynamics_bending_moment_free]
+    design = "C0TimoshenkoBeam.md"
```
Comment (recuero) [inline modules/tensor_mechanics/test/tests/shell/dynamics/tests:3]: This should be `ShellElement.md`

Comment (recuero) [reply modules/tensor_mechanics/test/tests/shell/dynamics/tests:3]: It was wrong to start with.

Comment (dschwen) [reply modules/tensor_mechanics/test/tests/shell/dynamics/tests:3]: oh, copy paste from the beam element verification?

**Comment by loganharbour** [comment]
@dschwen if you keep at this - feel free to up the required coverage for any modules you fixup here https://github.com/idaholab/moose/blob/next/.coverage

File: modules/tensor_mechanics/test/tests/shell/dynamics/tests (hunk 2 above)
Comment (recuero) [reply modules/tensor_mechanics/test/tests/shell/dynamics/tests]: Probably

**Review by recuero — APPROVED** [review]

**Review by cticenhour — APPROVED** [review]

**Pull request merged**